_site/index.html

```diff
@@ -518,7 +532,7 @@ <h3>T-Z</h3>
               <span class="word__alt">
                 <a class="word__link"
                   href="https://www.srhweek.ca/providers/people-and-communities/sizeism-and-fatphobia/">Sizeism and Fatphobia</a>;
-                <a class="word__link" href="https://www.huffpost.com/entry/fatphobia-a-guide-for-the_b_5214754">Fatphobia: A Guide for the Disbeliever</a>.
+                <a class="word__link" href="https://simmons.libguides.com/anti-oppression/anti-fatmisia">Anti-Oppression: Anti-Fatmisia</a>.
```

Comment:
I'm curious if you think we should to add an `Alt Words` section here to help illuminate the idea of "-phobia" versus "anti-" or "-misia" as we discussed.

Reply:
I think that would work, for sure, although "fatphobia" itself is an important word to know because the community uses it, plus we have so much literature that uses "fatphobia".

What do you think about it being more of a generalized entry that describes the nuance between those suffixes/prefixes, too, because it can apply to other entries within the dictionary?